### ISSUE TITLE
Argentina's flag issue

### ISSUE THREAD

**Issue by vfperri88** [LunaGao/flag_flutter#74]
Throw an error when selecting the AR code.

**Comment by cybex-dev** [comment]
Confirmed in `6.0.0`. 

`AR` is included in `flagsCode` found [here](https://github.com/LunaGao/flag_flutter/blob/stable/lib/flag_widget.dart#L113) and continues on [here](https://github.com/LunaGao/flag_flutter/blob/stable/lib/flag_widget.dart#L117) but returns an  blank/transparent widget.

**Comment by cybex-dev** [comment]
**Stacktrace:**

```
======== Exception caught by SVG ===================================================================
The following StateError was thrown resolving a single-frame picture stream:
Bad state: Expected to find Drawable with id url(#d).
Have ids: (url(#flag-icons-ar), url(#c), url(#a), url(#b), url(#h))

When the exception was thrown, this was the stack: 
#0      DrawableDefinitionServer.getDrawable (package:flutter_svg/src/vector_drawable.dart:579:7)
#1      _Elements.use (package:flutter_svg/src/svg/parser_state.dart:212:34)
#2      SvgParserState.parse (package:flutter_svg/src/svg/parser_state.dart:885:26)
<asynchronous suspension>
#3      SvgParser.parse (package:flutter_svg/parser.dart:27:12)
<asynchronous suspension>
#4      Svg.fromSvgString (package:flutter_svg/svg.dart:139:12)
<asynchronous suspension>
#5      Svg.svgPictureStringDecoder (package:flutter_svg/svg.dart:90:34)
<asynchronous suspension>
Picture provider: ExactAssetPicture(name: "packages/flag/res/4x3/ar.svg", bundle: null, colorFilter: null)
Picture key: AssetBundlePictureKey(bundle: PlatformAssetBundle#1ddfc(), name: "packages/flag/res/4x3/ar.svg", colorFilter: null, theme: SvgTheme(currentColor: null, fontSize: 14.0, xHeight: 7.0))
====================================================================================================
```

**Comment by cybex-dev** [comment]
**Update:** 
It seems multiple countries have issues loading flags.

Scrolling through the list of country codes found [here](https://github.com/Daniel-Ioannou/flutter_country_picker/blob/master/lib/src/res/country_codes.dart), I see there are some recognized countries whose flags do not show.

- Argentina (ar)
- Dominica (dm)
- Ecuador (ec)
- Grenada (gd)
- Haiti (ht)
- Malta (mt)
- Mexico (mx) [shows error but flag is present]
- Peru (pe) [shows error but flag is present]
- Serbia (rs)
- Sri Lanka (lk)

**Console output** - _sanitized for country codes and error outputs only_

```
I/flutter ( 8564): AF
I/flutter ( 8564): AX
I/flutter ( 8564): AL
I/flutter ( 8564): DZ
I/flutter ( 8564): AS
I/flutter ( 8564): AD
I/flutter ( 8564): AO
I/flutter ( 8564): AI
I/flutter ( 8564): AG
I/flutter ( 8564): AR
I/flutter ( 8564): AM
I/flutter ( 8564): AW
I/flutter ( 8564): AC
I/flutter ( 8564): AU
I/flutter ( 8564): AT
I/flutter ( 8564): AZ
I/flutter ( 8564): BS
I/flutter ( 8564): BH
I/flutter ( 8564): BD

======== Exception caught by SVG ===================================================================
The following StateError was thrown resolving a single-frame picture stream:
Bad state: Expected to find Drawable with id url(#d).
Have ids: (url(#flag-icons-ar), url(#c), url(#a), url(#b), url(#h))

When the exception was thrown, this was the stack: 
#0      DrawableDefinitionServer.getDrawable (package:flutter_svg/src/vector_drawable.dart:579:7)
#1      _Elements.use (package:flutter_svg/src/svg/parser_state.dart:212:34)
#2      SvgParserState.parse (package:flutter_svg/src/svg/parser_state.dart:885:26)
<asynchronous suspension>
#3      SvgParser.parse (package:flutter_svg/parser.dart:27:12)
<asynchronous suspension>
#4      Svg.fromSvgString (package:flutter_svg/svg.dart:139:12)
<asynchronous suspension>
#5      Svg.svgPictureStringDecoder (package:flutter_svg/svg.dart:90:34)
<asynchronous suspension>
Picture provider: ExactAssetPicture(name: "packages/flag/res/4x3/ar.svg", bundle: null, colorFilter: null)
Picture key: AssetBundlePictureKey(bundle: PlatformAssetBundle#86ab6(), name: "packages/flag/res/4x3/ar.svg", colorFilter: null, theme: SvgTheme(currentColor: null, fontSize: 14.0, xHeight: 7.0))
====================================================================================================
I/flutter ( 8564): AF
I/flutter ( 8564): AX
I/flutter ( 8564): AL
I/flutter ( 8564): DZ
I/flutter ( 8564): AS
I/flutter ( 8564): AD
I/flutter ( 8564): AO
I/flutter ( 8564): AI
I/flutter ( 8564): AG
I/flutter ( 8564): AR
I/flutter ( 8564): AM
I/flutter ( 8564): AW
I/flutter ( 8564): AC
I/flutter ( 8564): AU
I/flutter ( 8564): AT
I/flutter ( 8564): AZ
I/flutter ( 8564): BS
I/flutter ( 8564): BH
I/flutter ( 8564): BD
I/flutter ( 8564): BB
I/flutter ( 8564): BY
I/flutter ( 8564): BE
I/flutter ( 8564): BZ
I/flutter ( 8564): BJ
I/flutter ( 8564): BM
I/flutter ( 8564): BT
I/flutter ( 8564): BO
I/flutter ( 8564): BA
I/flutter ( 8564): BW
I/flutter ( 8564): BR
I/flutter ( 8564): IO
I/flutter ( 8564): VG
I/flutter ( 8564): BN
I/flutter ( 8564): BG
I/flutter ( 8564): BF
I/flutter ( 8564): BI
I/flutter ( 8564): KH
I/flutter ( 8564): CM
I/flutter ( 8564): CA
I/flutter ( 8564): CV
I/flutter ( 8564): BQ
I/flutter ( 8564): KY
I/flutter ( 8564): CF
I/flutter ( 8564): TD
I/flutter ( 8564): CL
I/flutter ( 8564): CN
I/flutter ( 8564): CX
I/flutter ( 8564): CC
I/flutter ( 8564): CO
I/flutter ( 8564): KM
I/flutter ( 8564): CD
I/flutter ( 8564): CG
I/flutter ( 8564): CK
I/flutter ( 8564): CR
I/flutter ( 8564): CI
I/flutter ( 8564): HR
I/flutter ( 8564): CU
I/flutter ( 8564): CW
I/flutter ( 8564): CY
I/flutter ( 8564): CZ
I/flutter ( 8564): DK
I/flutter ( 8564): DJ
I/flutter ( 8564): DM

======== Exception caught by SVG ===================================================================
The following FormatException was thrown resolving a single-frame picture stream:
Invalid double
1pt

When the exception was thrown, this was the stack: 
#0      double.parse (dart:core-patch/double_patch.dart:111:28)
#1      parseDouble (package:flutter_svg/src/utilities/numbers.dart:23:17)
#2      SvgParserState.parseDoubleWithUnits (package:flutter_svg/src/svg/parser_state.dart:1065:27)
#3      SvgParserState.parseStroke (package:flutter_svg/src/svg/parser_state.dart:1357:13)
#4      SvgParserState.parseStyle (package:flutter_svg/src/svg/parser_state.dart:1590:15)
#5      SvgParserState.addShape (package:flutter_svg/src/svg/parser_state.dart:955:7)
#6      SvgParserState.startElement (package:flutter_svg/src/svg/parser_state.dart:985:12)
#7      SvgParserState.parse (package:flutter_svg/src/svg/parser_state.dart:881:13)
<asynchronous suspension>
#8      SvgParser.parse (package:flutter_svg/parser.dart:27:12)
<asynchronous suspension>
#9      Svg.fromSvgString (package:flutter_svg/svg.dart:139:12)
<asynchronous suspension>
#10     Svg.svgPictureStringDecoder (package:flutter_svg/svg.dart:90:34)
<asynchronous suspension>
Picture provider: ExactAssetPicture(name: "packages/flag/res/4x3/dm.svg", bundle: null, colorFilter: null)
Picture key: AssetBundlePictureKey(bundle: PlatformAssetBundle#86ab6(), name: "packages/flag/res/4x3/dm.svg", colorFilter: null, theme: SvgTheme(currentColor: null, fontSize: 14.0, xHeight: 7.0))
====================================================================================================
I/flutter ( 8564): DO
I/flutter ( 8564): TL
I/flutter ( 8564): EC

======== Exception caught by SVG ===================================================================
The following FormatException was thrown resolving a single-frame picture stream:
Invalid double
1pt

When the exception was thrown, this was the stack: 
#0      double.parse (dart:core-patch/double_patch.dart:111:28)
#1      parseDouble (package:flutter_svg/src/utilities/numbers.dart:23:17)
#2      SvgParserState.parseDoubleWithUnits (package:flutter_svg/src/svg/parser_state.dart:1065:27)
#3      SvgParserState.parseStroke (package:flutter_svg/src/svg/parser_state.dart:1357:13)
#4      SvgParserState.parseStyle (package:flutter_svg/src/svg/parser_state.dart:1590:15)
#5      SvgParserState.addShape (package:flutter_svg/src/svg/parser_state.dart:955:7)
#6      SvgParserState.startElement (package:flutter_svg/src/svg/parser_state.dart:985:12)
#7      SvgParserState.parse (package:flutter_svg/src/svg/parser_state.dart:881:13)
<asynchronous suspension>
#8      SvgParser.parse (package:flutter_svg/parser.dart:27:12)
<asynchronous suspension>
#9      Svg.fromSvgString (package:flutter_svg/svg.dart:139:12)
<asynchronous suspension>
#10     Svg.svgPictureStringDecoder (package:flutter_svg/svg.dart:90:34)
<asynchronous suspension>
Picture provider: ExactAssetPicture(name: "packages/flag/res/4x3/ec.svg", bundle: null, colorFilter: null)
Picture key: AssetBundlePictureKey(bundle: PlatformAssetBundle#86ab6(), name: "packages/flag/res/4x3/ec.svg", colorFilter: null, theme: SvgTheme(currentColor: null, fontSize: 14.0, xHeight: 7.0))
====================================================================================================
I/flutter ( 8564): EG
I/flutter ( 8564): SV
I/flutter ( 8564): GQ
I/flutter ( 8564): ER
I/flutter ( 8564): EE
I/flutter ( 8564): SZ
I/flutter ( 8564): ET
I/flutter ( 8564): FK
I/flutter ( 8564): FO
I/flutter ( 8564): FJ
I/flutter ( 8564): FI
I/flutter ( 8564): FR
I/flutter ( 8564): GF
I/flutter ( 8564): PF
I/flutter ( 8564): GA
I/flutter ( 8564): GM
I/flutter ( 8564): GE
I/flutter ( 8564): DE
I/flutter ( 8564): GH
I/flutter ( 8564): GI
I/flutter ( 8564): GR
I/flutter ( 8564): GL
I/flutter ( 8564): GD

======== Exception caught by SVG ===================================================================
The following StateError was thrown resolving a single-frame picture stream:
Bad state: Expected to find Drawable with id url(#d).
Have ids: (url(#flag-icons-gd), url(#c), url(#b), url(#a))

When the exception was thrown, this was the stack: 
#0      DrawableDefinitionServer.getDrawable (package:flutter_svg/src/vector_drawable.dart:579:7)
#1      _Elements.use (package:flutter_svg/src/svg/parser_state.dart:212:34)
#2      SvgParserState.parse (package:flutter_svg/src/svg/parser_state.dart:885:26)
<asynchronous suspension>
#3      SvgParser.parse (package:flutter_svg/parser.dart:27:12)
<asynchronous suspension>
#4      Svg.fromSvgString (package:flutter_svg/svg.dart:139:12)
<asynchronous suspension>
#5      Svg.svgPictureStringDecoder (package:flutter_svg/svg.dart:90:34)
<asynchronous suspension>
Picture provider: ExactAssetPicture(name: "packages/flag/res/4x3/gd.svg", bundle: null, colorFilter: null)
Picture key: AssetBundlePictureKey(bundle: PlatformAssetBundle#86ab6(), name: "packages/flag/res/4x3/gd.svg", colorFilter: null, theme: SvgTheme(currentColor: null, fontSize: 14.0, xHeight: 7.0))
====================================================================================================
I/flutter ( 8564): GP
I/flutter ( 8564): GU
I/flutter ( 8564): GT
I/flutter ( 8564): GG
I/flutter ( 8564): GN
I/flutter ( 8564): GW
I/flutter ( 8564): GY
I/flutter ( 8564): HT

======== Exception caught by SVG ===================================================================
The following StateError was thrown resolving a single-frame picture stream:
Bad state: Expected to find Drawable with id url(#c).
Have ids: (url(#flag-icons-ht), url(#a), url(#b))

When the exception was thrown, this was the stack: 
#0      DrawableDefinitionServer.getDrawable (package:flutter_svg/src/vector_drawable.dart:579:7)
#1      _Elements.use (package:flutter_svg/src/svg/parser_state.dart:212:34)
#2      SvgParserState.parse (package:flutter_svg/src/svg/parser_state.dart:885:26)
<asynchronous suspension>
#3      SvgParser.parse (package:flutter_svg/parser.dart:27:12)
<asynchronous suspension>
#4      Svg.fromSvgString (package:flutter_svg/svg.dart:139:12)
<asynchronous suspension>
#5      Svg.svgPictureStringDecoder (package:flutter_svg/svg.dart:90:34)
<asynchronous suspension>
Picture provider: ExactAssetPicture(name: "packages/flag/res/4x3/ht.svg", bundle: null, colorFilter: null)
Picture key: AssetBundlePictureKey(bundle: PlatformAssetBundle#86ab6(), name: "packages/flag/res/4x3/ht.svg", colorFilter: null, theme: SvgTheme(currentColor: null, fontSize: 14.0, xHeight: 7.0))
====================================================================================================
I/flutter ( 8564): HM
I/flutter ( 8564): HN
I/flutter ( 8564): HK
I/flutter ( 8564): HU
I/flutter ( 8564): IS
I/flutter ( 8564): IN
I/flutter ( 8564): ID
I/flutter ( 8564): IR
I/flutter ( 8564): IQ
I/flutter ( 8564): IE
I/flutter ( 8564): IM
I/flutter ( 8564): IL
I/flutter ( 8564): IT
I/flutter ( 8564): JM
I/flutter ( 8564): JP
I/flutter ( 8564): JE
I/flutter ( 8564): JO
I/flutter ( 8564): KZ
I/flutter ( 8564): KE
I/flutter ( 8564): KI
I/flutter ( 8564): XK
I/flutter ( 8564): XK
I/flutter ( 8564): KW
I/flutter ( 8564): KG
I/flutter ( 8564): LA
I/flutter ( 8564): LV
I/flutter ( 8564): LB
I/flutter ( 8564): LS
I/flutter ( 8564): LR
I/flutter ( 8564): LY
I/flutter ( 8564): LI
I/flutter ( 8564): LT
I/flutter ( 8564): LU
I/flutter ( 8564): MO
I/flutter ( 8564): MK
I/flutter ( 8564): MG
I/flutter ( 8564): MW
I/flutter ( 8564): MY
I/flutter ( 8564): MV
I/flutter ( 8564): ML
I/flutter ( 8564): MT

======== Exception caught by SVG ===================================================================
The following FormatException was thrown resolving a single-frame picture stream:
Invalid double
1pt

When the exception was thrown, this was the stack: 
#0      double.parse (dart:core-patch/double_patch.dart:111:28)
#1      parseDouble (package:flutter_svg/src/utilities/numbers.dart:23:17)
#2      SvgParserState.parseDoubleWithUnits (package:flutter_svg/src/svg/parser_state.dart:1065:27)
#3      SvgParserState.parseStroke (package:flutter_svg/src/svg/parser_state.dart:1357:13)
#4      SvgParserState.parseStyle (package:flutter_svg/src/svg/parser_state.dart:1590:15)
#5      SvgParserState.addShape (package:flutter_svg/src/svg/parser_state.dart:955:7)
#6      SvgParserState.startElement (package:flutter_svg/src/svg/parser_state.dart:985:12)
#7      SvgParserState.parse (package:flutter_svg/src/svg/parser_state.dart:881:13)
<asynchronous suspension>
#8      SvgParser.parse (package:flutter_svg/parser.dart:27:12)
<asynchronous suspension>
#9      Svg.fromSvgString (package:flutter_svg/svg.dart:139:12)
<asynchronous suspension>
#10     Svg.svgPictureStringDecoder (package:flutter_svg/svg.dart:90:34)
<asynchronous suspension>
Picture provider: ExactAssetPicture(name: "packages/flag/res/4x3/mt.svg", bundle: null, colorFilter: null)
Picture key: AssetBundlePictureKey(bundle: PlatformAssetBundle#86ab6(), name: "packages/flag/res/4x3/mt.svg", colorFilter: null, theme: SvgTheme(currentColor: null, fontSize: 14.0, xHeight: 7.0))
====================================================================================================
I/flutter ( 8564): MH
I/flutter ( 8564): MQ
I/flutter ( 8564): MR
I/flutter ( 8564): MU
I/flutter ( 8564): YT
I/flutter ( 8564): MX
I/flutter ( 8564): FM
I/flutter ( 8564): MD
I/flutter ( 8564): MC
I/flutter ( 8564): MN
I/flutter ( 8564): ME
I/flutter ( 8564): MS
I/flutter ( 8564): MA

======== Exception caught by SVG ===================================================================
The following assertion was thrown while parsing AssetBundlePictureKey(bundle: PlatformAssetBundle#86ab6(), name: "packages/flag/res/4x3/mx.svg", colorFilter: null, theme: SvgTheme(currentColor: null, fontSize: 14.0, xHeight: 7.0)) in radialGradient:
Failed to find definition for #a

This library only supports <defs> and xlink:href references that are defined ahead of their references.

This error can be caused when the desired definition is defined after the element referring to it (e.g. at the end of the file), or defined in another file.

This error is treated as non-fatal, but your SVG file will likely not render as intended
====================================================================================================

======== Exception caught by SVG ===================================================================
The following assertion was thrown while parsing AssetBundlePictureKey(bundle: PlatformAssetBundle#86ab6(), name: "packages/flag/res/4x3/mx.svg", colorFilter: null, theme: SvgTheme(currentColor: null, fontSize: 14.0, xHeight: 7.0)) in radialGradient:
Failed to find definition for #a

This library only supports <defs> and xlink:href references that are defined ahead of their references.

This error can be caused when the desired definition is defined after the element referring to it (e.g. at the end of the file), or defined in another file.

This error is treated as non-fatal, but your SVG file will likely not render as intended
====================================================================================================

======== Exception caught by SVG ===================================================================
The following assertion was thrown while parsing AssetBundlePictureKey(bundle: PlatformAssetBundle#86ab6(), name: "packages/flag/res/4x3/mx.svg", colorFilter: null, theme: SvgTheme(currentColor: null, fontSize: 14.0, xHeight: 7.0)) in radialGradient:
Failed to find definition for #a

This library only supports <defs> and xlink:href references that are defined ahead of their references.

This error can be caused when the desired definition is defined after the element referring to it (e.g. at the end of the file), or defined in another file.

This error is treated as non-fatal, but your SVG file will likely not render as intended
====================================================================================================
I/flutter ( 8564): MZ
I/flutter ( 8564): MM
I/flutter ( 8564): NA
I/flutter ( 8564): NR
I/flutter ( 8564): NP
I/flutter ( 8564): NL
I/flutter ( 8564): NC
I/flutter ( 8564): NZ
I/flutter ( 8564): NI
I/flutter ( 8564): NE
I/flutter ( 8564): NG
I/flutter ( 8564): NU
I/flutter ( 8564): NF
I/flutter ( 8564): KP
I/flutter ( 8564): MP
I/flutter ( 8564): NO
I/flutter ( 8564): OM
I/flutter ( 8564): PK
I/flutter ( 8564): PW
I/flutter ( 8564): PS
I/flutter ( 8564): PA
I/flutter ( 8564): PG
I/flutter ( 8564): PY
I/flutter ( 8564): PE
I/flutter ( 8564): PH
I/flutter ( 8564): PL
I/flutter ( 8564): PT
I/flutter ( 8564): PR
I/flutter ( 8564): QA
I/flutter ( 8564): RE
I/flutter ( 8564): RO
I/flutter ( 8564): RU
I/flutter ( 8564): RW
I/flutter ( 8564): BL
I/flutter ( 8564): SH
I/flutter ( 8564): KN

======== Exception caught by SVG ===================================================================
The following StateError was thrown resolving a single-frame picture stream:
Bad state: Expected to find Drawable with id url(#c).
Have ids: (url(#flag-icons-pe), url(#g), url(#d), url(#a), url(#b), url(#e), url(#f))

When the exception was thrown, this was the stack: 
#0      DrawableDefinitionServer.getDrawable (package:flutter_svg/src/vector_drawable.dart:579:7)
#1      _Elements.use (package:flutter_svg/src/svg/parser_state.dart:212:34)
#2      SvgParserState.parse (package:flutter_svg/src/svg/parser_state.dart:885:26)
<asynchronous suspension>
#3      SvgParser.parse (package:flutter_svg/parser.dart:27:12)
<asynchronous suspension>
#4      Svg.fromSvgString (package:flutter_svg/svg.dart:139:12)
<asynchronous suspension>
#5      Svg.svgPictureStringDecoder (package:flutter_svg/svg.dart:90:34)
<asynchronous suspension>
Picture provider: ExactAssetPicture(name: "packages/flag/res/4x3/pe.svg", bundle: null, colorFilter: null)
Picture key: AssetBundlePictureKey(bundle: PlatformAssetBundle#86ab6(), name: "packages/flag/res/4x3/pe.svg", colorFilter: null, theme: SvgTheme(currentColor: null, fontSize: 14.0, xHeight: 7.0))
====================================================================================================
I/flutter ( 8564): LC
I/flutter ( 8564): MF
I/flutter ( 8564): PM
I/flutter ( 8564): VC
I/flutter ( 8564): WS
I/flutter ( 8564): SM
I/flutter ( 8564): ST
I/flutter ( 8564): SA
I/flutter ( 8564): SN
I/flutter ( 8564): RS
I/flutter ( 8564): SC
I/flutter ( 8564): SL
I/flutter ( 8564): SG
I/flutter ( 8564): SX
I/flutter ( 8564): SK
I/flutter ( 8564): SI
I/flutter ( 8564): SB

======== Exception caught by SVG ===================================================================
The following StateError was thrown resolving a single-frame picture stream:
Bad state: Expected to find Drawable with id url(#c).
Have ids: (url(#flag-icons-rs), url(#b), url(#d))

When the exception was thrown, this was the stack: 
#0      DrawableDefinitionServer.getDrawable (package:flutter_svg/src/vector_drawable.dart:579:7)
#1      _Elements.use (package:flutter_svg/src/svg/parser_state.dart:212:34)
#2      SvgParserState.parse (package:flutter_svg/src/svg/parser_state.dart:885:26)
<asynchronous suspension>
#3      SvgParser.parse (package:flutter_svg/parser.dart:27:12)
<asynchronous suspension>
#4      Svg.fromSvgString (package:flutter_svg/svg.dart:139:12)
<asynchronous suspension>
#5      Svg.svgPictureStringDecoder (package:flutter_svg/svg.dart:90:34)
<asynchronous suspension>
Picture provider: ExactAssetPicture(name: "packages/flag/res/4x3/rs.svg", bundle: null, colorFilter: null)
Picture key: AssetBundlePictureKey(bundle: PlatformAssetBundle#86ab6(), name: "packages/flag/res/4x3/rs.svg", colorFilter: null, theme: SvgTheme(currentColor: null, fontSize: 14.0, xHeight: 7.0))
====================================================================================================
I/flutter ( 8564): SO
I/flutter ( 8564): ZA
I/flutter ( 8564): GS
I/flutter ( 8564): KR
I/flutter ( 8564): SS
I/flutter ( 8564): ES
I/flutter ( 8564): LK
I/flutter ( 8564): SD
I/flutter ( 8564): SR

======== Exception caught by SVG ===================================================================
The following StateError was thrown resolving a single-frame picture stream:
Bad state: Expected to find Drawable with id url(#b).
Have ids: (url(#flag-icons-lk), url(#a))

When the exception was thrown, this was the stack: 
#0      DrawableDefinitionServer.getDrawable (package:flutter_svg/src/vector_drawable.dart:579:7)
#1      _Elements.use (package:flutter_svg/src/svg/parser_state.dart:212:34)
#2      SvgParserState.parse (package:flutter_svg/src/svg/parser_state.dart:885:26)
<asynchronous suspension>
#3      SvgParser.parse (package:flutter_svg/parser.dart:27:12)
<asynchronous suspension>
#4      Svg.fromSvgString (package:flutter_svg/svg.dart:139:12)
<asynchronous suspension>
#5      Svg.svgPictureStringDecoder (package:flutter_svg/svg.dart:90:34)
<asynchronous suspension>
Picture provider: ExactAssetPicture(name: "packages/flag/res/4x3/lk.svg", bundle: null, colorFilter: null)
Picture key: AssetBundlePictureKey(bundle: PlatformAssetBundle#86ab6(), name: "packages/flag/res/4x3/lk.svg", colorFilter: null, theme: SvgTheme(currentColor: null, fontSize: 14.0, xHeight: 7.0))
====================================================================================================
I/flutter ( 8564): SJ
I/flutter ( 8564): SE
I/flutter ( 8564): CH
I/flutter ( 8564): SY
I/flutter ( 8564): TW
I/flutter ( 8564): TJ
I/flutter ( 8564): TZ
I/flutter ( 8564): TH
I/flutter ( 8564): TG
I/flutter ( 8564): TK
I/flutter ( 8564): TO
I/flutter ( 8564): TT
I/flutter ( 8564): TN
I/flutter ( 8564): TR
I/flutter ( 8564): TM
I/flutter ( 8564): TC
I/flutter ( 8564): TV
I/flutter ( 8564): VI
I/flutter ( 8564): UG
I/flutter ( 8564): UA
I/flutter ( 8564): AE
I/flutter ( 8564): GB
I/flutter ( 8564): US
I/flutter ( 8564): UY
I/flutter ( 8564): UZ
I/flutter ( 8564): VU
I/flutter ( 8564): VA
I/flutter ( 8564): VE
I/flutter ( 8564): VN
I/flutter ( 8564): WF
I/flutter ( 8564): EH
I/flutter ( 8564): YE
I/flutter ( 8564): ZM
I/flutter ( 8564): ZW
```

**Comment by Yahllil** [comment]
I experience this issue with the USA flag. Any update?

**Comment by divan** [comment]
I have the same issue with Lithuania flag (lt)

**Comment by divan** [comment]
Seems like something's broken with flutter_svg on Flutter 3 :/
https://github.com/dnfield/flutter_svg/issues

**Comment by FernandoSini** [comment]
I'm currently using flags on version 5.0.1 while version 6.0 has some bugs. The version 5.0.1 is currently working normally with flutter 3 for who is having issue with version 6.0. For who has issue, I recommend use version 5.0.1 while the version  6.0 of this package currently has issues.

**Comment by XiXiongMaoXiong** [comment]
> I'm currently using flags on version 5.0.1 while version 6.0 has some bugs. The version 5.0.1 is currently working normally with flutter 3 for who is having issue with version 6.0. For who has issue, I recommend use version 5.0.1 while the version 6.0 of this package currently has issues.

This is the only solution as of now, because v6.x also fails for me for literally every single flag.

**Comment by fmdogan** [comment]
Updating flutter_svg package may help but Flag restricts svg version with maximum 1.1.0 as i get error on pub get. It should be fixed.. I do not want to downgrade to 5x version because it has size issue for some flags.

the countries which fails on my app (flag: v6.0.0, flutter 3.7.7):
Argentina (ar),
Peru (pe),
Serbia (rs),
Sri Lanka (lk),
Haiti (ht),
Grenada (gd)

**Comment by LunaGao** [comment]
![image](https://github.com/LunaGao/flag_flutter/assets/5066920/a731db27-98fa-46a9-95b8-21f583622db4)
Argentina (ar),
Peru (pe),
Serbia (rs),
Sri Lanka (lk),
Haiti (ht),
Grenada (gd)

those has been fixed in 7.0.0, and it already published on pub.dev.
